defaults/config.js

```diff
@@ -35,6 +35,13 @@ module.exports = {
 	// This value is set to `9000` by default.
 	port: 9000,
 
+	// ### `chmod`
+	//
+	// Set the permissions of the UNIX domain socket.
+	//
+	// This value is set to `0o660` by default.
+	chmod: 0o660,
```

Comment:
chmod doesn't seem to be a good name for it in my view... chmod is the command that changes permissions, not the name of the thing.

socketPermissions would by my preference, as that's what it is. Also needs an easy to understand comment that it only applies if the host option is a unix socket and ignored otherwise